src/test/K6/src/tests/platform/eFormidling/eFormidling.js

```diff
@@ -0,0 +1,67 @@
+/*
+    Command: docker-compose run k6 run /src/tests/platform/eformidling/eformidling.js -e env=*** -e operatoraccesskey=*** -e messageId=*** -e orgNo=***
+*/
+import { check } from 'k6';
+import { addErrorCount } from '../../../errorcounter.js';
+import * as eFormidling from '../../../api/platform/eFormidling/eformidling.js'
+import { generateJUnitXML, reportPath } from '../../../report.js';
+
+const messageId = __ENV.messageId;
+const orgNo = __ENV.orgNo;
+
+export const options = {
+  thresholds: {
+    errors: ['count<1'],
+  },
+};
+
+export default function(){
+  var res = eFormidling.getStatuses('');
+  var success = check(res, {
+    'Get Statuses without messageId Status is 400':(r) => r.status === 400,
+    'Response body informs what is missing': (r) => r.json("message").includes('Query parameter messageId is required')
+  });
+
+  addErrorCount(success);
+
+  var res = eFormidling.getStatuses(messageId);
+  var success = check(res, {
+    'Get Statuses with messageId Status is 200':(r) => r.status === 200
+  });
+  addErrorCount(success);
+
+  var res = eFormidling.getConversation('');
+  var success = check(res, {
+    'Get Conversations without messageId Status is 400':(r) => r.status === 400,
+    'Response body informs what is missing': (r) => r.json("message").includes('Query parameter messageId is required')
```

Comment:
suggest to modify the name of the test. It will be confusing to know which test actually failed when the test results are merged when run from azdo